doc/source/data/inspecting-data.rst

```diff
@@ -0,0 +1,130 @@
+.. _inspecting-data:
+
+===============
+Inspecting data
+===============
+
+.. _describing-datasets:
+
+Describing datasets
+===================
+
+:class:`Datasets <ray.data.Dataset>` are tabular. To view a Dataset's column names and
+types, call :meth:`Dataset.schema() <ray.data.Dataset.schema>`.
+
+.. testcode::
+
+    import ray
+
+    ds = ray.data.read_csv("s3://anonymous@air-example-data/iris.csv")
+
+    print(ds.schema())
+
+.. testoutput::
+
+    Column             Type
+    ------             ----
+    sepal length (cm)  double
+    sepal width (cm)   double
+    petal length (cm)  double
+    petal width (cm)   double
+    target             int64
+
+For more information such as the number of rows, print the Dataset.
+
+.. testcode::
+
+    import ray
+
+    ds = ray.data.read_csv("s3://anonymous@air-example-data/iris.csv")
+
+    print(ds)
+
+.. testoutput::
+
+    Dataset(
+       num_blocks=...,
+       num_rows=150,
+       schema={
+          sepal length (cm): double,
+          sepal width (cm): double,
+          petal length (cm): double,
+          petal width (cm): double,
+          target: int64
+       }
+    )
+
+.. _inspecting-rows:
+
+Inspecting rows
+===============
+
+To inspect rows, call :meth:`Dataset.take() <ray.data.Dataset.take>` or
```

Comment:
Just add a little bit more on what this is doing? To inspect rows, call ..., which returns a list of rows from the Dataset.